src/modules/fancyzones/editor/FancyZonesEditor/EditorWindow.cs

```diff
@@ -16,6 +16,11 @@ protected void OnSaveApplyTemplate(object sender, RoutedEventArgs e)
             EditorOverlay mainEditor = EditorOverlay.Current;
             if (mainEditor.DataContext is LayoutModel model)
             {
+                if (model.Type == LayoutType.Blank)
+                {
+                    model.Type = LayoutType.Custom;
```

Comment:
Maybe some explanation for this if statement, not as a comment but just here in pull request, so it would be easier to understand.

Reply:
I'll add the comment

Reply:
Done